python/cudf/dataframe/multiindex.py

```diff
@@ -218,6 +271,8 @@ def is_contiguous(self):
 
     @property
     def size(self):
+        if self._codes is None:
+            self._compute_levels_and_codes()
         return len(self.codes[0])
```

Comment:
Instead of the codes here you could just get the size of the columns so we don't need the codes

Reply:
Agree